spotbugsTestCases/src/java/lambdas/Issue2436.java

```diff
@@ -0,0 +1,10 @@
+package lambdas;
```

Comment:
This issue is not about lambas, can you please move this file to the appropriate package? If you can't find better, then `ghIssues` would be okay. (Don't forget to modify the path in the test file as well.)